backend/tracim_backend/__init__.py

```diff
@@ -235,9 +236,12 @@ def web(global_config: OrderedDict, **local_settings) -> Router:
     # make sure to add this before other routes to intercept OPTIONS
     configurator.add_cors_preflight_handler()
     # Default authorization : Accept anything.
-    configurator.set_authorization_policy(AcceptAllAuthorizationPolicy())
-    authn_policy = MultiAuthenticationPolicy(policies)
-    configurator.set_authentication_policy(authn_policy)
+    configurator.set_security_policy(
+        TracimSecurityPolicy(
+            authn_policy=MultiAuthenticationPolicy(policies),
```

Comment:
Note: the pyramid_multiauth is not up to date for pyramid v2, so we still use the old authentication policy logic (deprecated) there.